include/fmt/core.h

```diff
@@ -502,6 +505,7 @@ using string_view = basic_string_view<char>;
 template <typename T> struct is_char : std::false_type {};
 template <> struct is_char<char> : std::true_type {};
```

Comment:
These are also undocumented and semi-internal so let's just export `basic_string_view` via `FMT_MODULE_EXPORT` (we don't need the block here).

Reply:
The primary template needs to be exported. It is documented to be [specializable by users](https://fmt.dev/latest/api.html#wchar-t-support). The (library-internal) specializations don't need to be exported as long as they are reachable (ehich they are).

Reply:
My bad. We should probably make it private but not in this PR.